### DISCUSSION TITLE
icon request: kangaroo

### DISCUSSION THREAD

**Discussion by blinossier** [FortAwesome/Font-Awesome#21166] [Icon Requests]
Kangaroo icon would be great
![image kangaroo](https://cloud.githubusercontent.com/assets/23188514/19887169/0621baf6-a07b-11e6-9d07-51f24d9f6996.jpg)

**Comment by adhoc-jan** [comment]
+1

**Comment by esglobal** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🦘 [U+1F998](https://emojipedia.org/emoji/%F0%9F%A6%98/) Kangaroo

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by naher94** [comment]
I too, will upvote `fa-kangaroo`
